frontend/src/APIClients/CampsAPIClient.ts

```diff
@@ -36,7 +36,19 @@ const editCampById = async (
   }
 };
 
+const deleteCamp = async (id: string): Promise<boolean> => {
+  try {
+    await baseAPIClient.delete(`/camp/${id}`, {
+      headers: { Authorization: BEARER_TOKEN },
```

Comment:
For cleanliness - can we create method in the `CampsAPIClient` for this?

Reply:
What do you mean by this?

Reply:
Oh never mind, I don't know what I thought I saw ❓ 

This looks good